{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,75 @@
+
+[![PyPI](https://img.shields.io/pypi/v/{{cookiecutter.project_name}}.svg)][pypi_]
+[![Status](https://img.shields.io/pypi/status/{{cookiecutter.project_name}}.svg)][status]
+[![Python Version](https://img.shields.io/pypi/pyversions/{{cookiecutter.project_name}})][python version]
+[![License](https://img.shields.io/pypi/l/{{cookiecutter.project_name}})][license]
+
+[![Read the documentation at https://{{cookiecutter.project_name}}.readthedocs.io/](https://img.shields.io/readthedocs/{{cookiecutter.project_name}}/latest.svg?label=Read%20the%20Docs)][read the docs]
+[![Tests](https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/workflows/Tests/badge.svg)][tests]
+[![Codecov](https://codecov.io/gh/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/branch/main/graph/badge.svg)][codecov]
+
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)][pre-commit]
+[![Black](https://img.shields.io/badge/code%20style-black-000000.svg)][black]
+
+[pypi_]: https://pypi.org/project/{{cookiecutter.project_name}}/
+[status]: https://pypi.org/project/{{cookiecutter.project_name}}/
+[python version]: https://pypi.org/project/{{cookiecutter.project_name}}
+[license]: https://opensource.org/licenses/{{cookiecutter.license}}
+[read the docs]: https://{{cookiecutter.project_name}}.readthedocs.io/
+[tests]: https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/actions?workflow=Tests
+[codecov]: https://app.codecov.io/gh/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}
+[pre-commit]: https://github.com/pre-commit/pre-commit
+[black]: https://github.com/psf/black
+
+## Features
+
+- TODO
+
+## Requirements
+
+- TODO
+
+## Installation
+
+You can install _{{cookiecutter.friendly_name}}_ via [pip] from [PyPI]:
+
+```console
+$ pip install {{cookiecutter.project_name}}
+```
+
+## Usage
+
+Please see the [Command-line Reference] for details.
+
+## Contributing
+
+Contributions are very welcome.
+To learn more, see the [Contributor Guide].
+
+## License
+
+Distributed under the terms of the [{{cookiecutter.license.replace("-", " ")}} license],
+_{{cookiecutter.friendly_name}}_ is free and open source software.
+
+## Issues
+
+If you encounter any problems,
+please [file an issue] along with a detailed description.
+
+## Credits
+
+This project was generated from [@cjolowicz]'s [Hypermodern Python Cookiecutter] template.
+
+[@cjolowicz]: https://github.com/cjolowicz
+[cookiecutter]: https://github.com/audreyr/cookiecutter
+[{{cookiecutter.license.replace("-", " ").lower()}} license]: https://opensource.org/licenses/{{cookiecutter.license}}
+[pypi]: https://pypi.org/
+[hypermodern python cookiecutter]: https://github.com/cjolowicz/cookiecutter-hypermodern-python
+[file an issue]: https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/issues
+[pip]: https://pip.pypa.io/
+
+<!-- github-only -->
+
+[contributor guide]: CONTRIBUTING.md
```

Comment:
This cannot be a relative link. It works on GitHub, but not on the PyPI project page.

```suggestion
[contributor guide]: [CONTRIBUTING.md](https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/blob/main/CONTRIBUTING.md)
```

Apart from this, this is ready to be merged 💯

Reply:
Fixed that! @cjolowicz